### PR TITLE
Feat time ago pipe

### PR DESCRIPTION
**Pull request by christopheelkhoury**
Solves #102 

Changes Proposed in this Pull Request:
- Create `time-ago.ts` and `time-ago.spec.ts` pipes inside a `date` sub-directory
- This pipe will take a `Date` object as an input and transform it into a `X _number of date unit_ _date unit_(s) ago`

As per the following table:

Time Range | Output
-- | --
0 - 59 seconds | just now
1 minute | a minute ago
2 - 59 minutes | X minutes ago
1 hour | 1 hour ago
2 - 23 hours | X hours ago
1 day | yesterday
2 - 6 days | X days ago
1 week | last week
2 - 3 weeks | X weeks ago
1 month | last month
2 - 11 months | X months ago
1 year | last year
1 + years | X years ago

### CODE REVIEW

File: src/pipes/date/time-ago.spec.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import { TimeAgoPipe } from "./time-ago";
+
+fdescribe("TimeAgoPipe", () => {
```
Comment (danrevah) [inline src/pipes/date/time-ago.spec.ts]: change this to describe it will fail the tests

File: src/pipes/date/time-ago.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({ name: 'time-ago' })
+export class TimeAgoPipe implements PipeTransform {
+
```
Comment (danrevah) [inline src/pipes/date/time-ago.ts]: too many spaces

File: src/pipes/date/time-ago.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+      return days === 1 ? 'yesterday' : days + ' days ago';
+    } else if (hours > 0) {
+      return hours === 1 ? 'an hour ago' : hours + ' hours ago';
+    } else if (minutes > 0) {
+      return minutes === 1 ? 'just now' : minutes + ' minutes ago';
+    } else {
+      return 'just now';
+    }
+  }
+}
```
Comment (danrevah) [inline src/pipes/date/time-ago.ts]: Can you add a timer like in `ngx-moment` that will handle changes if the page keeps open?

Comment (christopheelkhoury) [reply src/pipes/date/time-ago.ts]: I didn't understand your comment. Can you please elaborate?

**Comment by danrevah** [comment]
Hey @christopheelkhoury,
I've made some adjustments to your code and added it to `v2.4.5`.

- Fix the `fdescribe` issue
- Code refactor for readability 
- Added support to moment.js

Thank you for your contribution!

**Pull request merged**